README.md

```diff
@@ -269,6 +269,14 @@ profile.registerType(classToRegister, locationToRegister, 'container', 'listed')
     // querying the registry now will include the new container
     profile.typeRegistryForClass(vocab.sioc('Post'))
   })
+
+// To remove the same class from registry:
+var classToRemove = vocab.sioc('Post')
+profile.unregisterType(classToRemove, 'listed')
```

Comment:
My feeling is that if the registry is not public, then you don't need the extra param in the function.